Makefile

```diff
@@ -28,8 +28,12 @@ core_api_gen:
 	make mock_astro_core
 
 test:
-	go test -count=1 -cover $(shell go list ./... | grep -v e2e)
-	go test -coverprofile=coverage.txt -covermode=atomic $(shell go list ./... | grep -v e2e)
+	#go test -count=1 -cover $(shell go list ./... | grep -v e2e)
+	#go test -coverprofile=coverage.txt -covermode=atomic $(shell go list ./... | grep -v e2e)
+	go mod download
+	go install github.com/onsi/ginkgo/ginkgo@v1.12.0
```

Comment:
nit: any reason not to use the latest version? Also, would it make sense to install it in the local `bin` folder instead of the global install?

Reply:
@neel-astro the reason we aren't using the latest version is that when we use the latest version it updates other libraries in a breaking fashion (looks like k8s does breaking changes on minor version bumps)